src/dynamicprompts/samplers/base.py

```diff
@@ -15,10 +15,12 @@ def __init__(
         self,
         *,
         wildcard_manager: WildcardManager,
+        parser_config: ParserConfig,
```

Comment:
Same here... I think this should use the default parser config, so it doesn't need to be explicitly passed in.